src/model.d.ts

```diff
@@ -570,6 +570,39 @@ export interface WhereOperators<AttributeType = any> {
    * https://www.postgresql.org/docs/14/functions-range.html
    */
   [Op.adjacent]?: WhereOperators<AttributeType>[typeof Op.strictLeft];
+
+  /**
+   * PG only
+   *
+   * Check if any of these array strings exist as top-level keys.
+   *
+   * @example
+   * ```typescript
+   * { jsonbAttribute: { [Op.anyKeyExists]: ['a','b'] } }
+   * // results in
+   * // "jsonbAttribute" ?| ARRAY['a','b']
+   * ```
+   *
+   * https://www.postgresql.org/docs/current/functions-json.html
+   */
+  [Op.anyKeyExists]?: Array<NonNullable<string>>
```

Comment:
string is already non-nullable

```suggestion
  [Op.anyKeyExists]?: Array<string>
```